src/app/domains/profiles/profiles.helpers.tsx

```diff
@@ -21,3 +21,23 @@ export function useGetProfiles() {
     ...rest,
   }
 }
+
+export function useUpdateUserConnection() {
+  const { supabase } = useSupabase()
+
+  const mutation = useMutation(['updateUserConnection'], async () => {
+    const { data } = await supabase.auth.getSession()
+    await supabase
+      .from('profiles')
+      .update({
+        timestamp_last_connection: new Date().toISOString().toLocaleString(),
+      })
+      .eq('id', data.session?.user.id)
```

Comment:
question(security): Could I hypothetically update the timestamp or any other person?

Reply:
good question hehe, RLS is enabled on the profiles table, so users can only update their own profile

Reply:
perfect then ^.^